src/site/content/en/metrics/user-centric-performance-metrics/index.md

```diff
@@ -0,0 +1,194 @@
+    <td>Did the navigation start successfully? Has the server responded?</td>
+  </tr>
+  <tr>
+    <td><strong>Is it useful?</strong></td>
+    <td>Has enough content rendered that users can engage with it?</td>
+  </tr>
+  <tr>
+    <td><strong>Is it usable?</strong></td>
+    <td>Can users interact with the page, or is it busy?</td>
+  </tr>
+  <tr>
+    <td><strong>Is it delightful?</strong></td>
+    <td>Are the interactions smooth and natural, free of lag and jank?</td>
+  </tr>
+</table>
+
+## How metrics are measured
+
+Performance metrics are generally measured in one of two ways:
+
+- **In the lab:** using tools to simulate a page load in a consistent,
+  controlled environment.
+- **In the field**: on real users actually loading and interacting with the page
+
+Neither of these options is necessarily better or worse than the other&mdash;in
+fact you generally want to use both to ensure good performance.
+
+### In the lab
+
+Testing performance in the lab is essential when developing new features. Before
+features are released in production, it's impossible to measure their
+performance characteristics on real users, so testing them in the lab before the
+feature is released is the best way to prevent performance regressions.
+
+### In the field
+
+On the other hand, while testing in the lab is a reasonable proxy for
+performance, it isn't necessarily reflective of how all users experience your
+site in the wild.
+
+The performance of a site can vary dramatically based on a user's device
+capabilities and their network conditions. It can also vary based on whether (or
+how) a user is interacting with the page.
+
+Moreover, page loads may not be deterministic. For example, sites that load
+personalized content or ads may experience vastly different performance
+characteristics from user to user. A lab test will not capture those
+differences.
+
+The only way to truly know how your site performs for your users is to actually
+measure its performance as those users are loading and interacting with it. This
+type of measurement is commonly referred to as [Real User
+Monitoring](https://en.wikipedia.org/wiki/Real_user_monitoring)&mdash;or RUM for
+short.
+
+## Types of metrics
+
+There are several other types of metrics that are relevant to how users perceive
+performance.
+
+- **Perceived load speed:** how quickly a page can load and render all of its
+  visual elements to the screen.
+- **Load responsiveness:** how quickly a page can load and execute any required
+  JavaScript code in order for components to respond quickly to user interaction
+- **Runtime responsiveness:** after page load, how quickly can the page respond
+  to user interaction.
+- **Visual stability:** do elements on the page shift in ways that users don't
+  expect and potentially interfere with their interactions?
+- **Smoothness:** do transitions and animations render at a consistent frame
+  rate and flow fluidly from one state to the next?
```

Comment:
I'd suggest consistent wording for these (question vs not, always/never using ending marks in the bullet points of this list, etc.). If using wording similar to the first one, consider something like 'how quickly visual elements of a page are rendered' (less awkwardly worded IMO).

Reply:
Thanks for the suggestion. I think we'll address this in a separate PR since this doc was actually part of #1854. (sorry the base branch has moved around, so that wasn't clear).